CesiumGS/cesium-vite-example
# Remove node externals

With the release of [CesiumJS `v1.114`](https://github.com/CesiumGS/cesium/releases/tag/1.114) which included https://github.com/CesiumGS/cesium/pull/11773 the external node modules are no longer necessary and can be excluded from the config. This pr removes them.

@ggetz Do you think it makes sense to still include a comment about this in the readme or somewhere? Or just remove and only support the latest version of cesium?

## Testing

- clean your local project
- run `npm install`
- run `npm run dev` and check the local version works
- run `npm run build` and make sure there are no errors
- run `npm run preview` and make sure the local built version still works
